src/components/base/JInputField/JInputField.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+
+import { getErrorMessage } from 'utils/form'
+import { JFieldMessage } from 'components/base'
+
+import jInputFieldStyle from './jInputField.m.scss'
+
+export type JInputType = 'text' | 'password'
+export type JInputTheme = 'white'
+
+type Props = StyledComponent<JInputTheme> & {
+  disabled: boolean,
+  infoMessage: string,
+  input: FinalFormInput,
+  label: string,
+  meta: FinalFormMeta,
+  placeholder: string,
+  type: JInputType,
+  validateType: FinalFormValidateType,
+}
+
+function JInputField({
+  className,
+  theme,
+  disabled,
+  infoMessage,
+  input,
+  label,
+  meta,
+  placeholder,
+  type,
+  validateType,
+  ...rest
+}: Props) {
+  const textInput = React.createRef()
+
+  const errorMessage = getErrorMessage(meta, validateType)
+  const hasLabel = !!label
+  const hasPlaceholder = !!placeholder
+  const hasError = !!errorMessage
+  const hasInfo = !!infoMessage
+  const hasValue = !!input.value
+  const isActive = meta.active || hasValue || (hasLabel && hasPlaceholder)
+
+  return (
+    <div
+      className={classNames(
+        jInputFieldStyle.core,
+        jInputFieldStyle[theme],
+        className,
+      )}
+      onClick={() => textInput.current.focus()}
```

Comment:
It's not good idea: create new function in render.
I guess, the better way is
```
onClick={textInput.current.focus}
```

Reply:
[eslint] Handler function for onClick prop key must begin with 'handle' (react/jsx-handler-names)
This will fail with this eslint error

Reply:
This is an error. You should definitely not pass new function to HTML element on every render. You could do it in several ways so there should be no ESLint error, or you could turn ESLint rule off, but in general we prefer writing error-prone code to writing code exactly according to rules.

Reply:
Fixed in this commit: https://github.com/jibrelnetwork/jwallet-web/pull/717/commits/1a0a767028c60ac2c2def854a76795335010b1b6